src/SingleProject/Resizetizer/src/SkiaSharpAppIconTools.cs

```diff
@@ -83,6 +83,13 @@ void Draw(SKBitmap tempBitmap, DpiPath dpi, SKSize unscaledCanvasSize)
 
 			using var canvas = new SKCanvas(tempBitmap);
 
+			if (GetClipPath(dpi, canvasSize, unscaledCanvasSize) is { } clipPath)
+			{
+				canvas.Clear(SKColors.Transparent);
+
+				canvas.ClipPath(clipPath, antialias: true);
+			}
```

Comment:
This is the real code change, if there is a clip path attached to the "dpi" then we clip to that first.